example_school_library_decorator/main.rb

```diff
@@ -102,13 +79,19 @@ def main
 
     case response
     when "1"
-      bookstore.list_all_books()
+      Book.list_all_books()
     when "2"
       Person.list_all_people()
     when "3"
       Person.create_person()
     when "4"
-      bookstore.create_book()
+      print "Book title: "
+      title = gets.chomp
+      print "Book author: "
+      author = gets.chomp
+      book = Book.new(title, author)
+      Book.add_books(book)
+      puts "Book created succesfully"  
```

Comment:
- [ ] I noticed that you have not applied the Single Responsibility Principle(Read More about this [here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid)) in your switch case. As mentioned in my other comments, you are internally storing Books and People in their classes when they should be stored withing your `Bookstore || App class`. These classes should only classes that we create instances from which will be stored in our `Bookstore || App class`. That way we have this class single running all operations it is required to do. 
     -  Kindly transfer all methods of listing and creation of @rentals, @books and @people to the `Bookstore || App` class as it is its main responsibility.
     - Kindly also ensure that prompts for input should not be within classes like` Book or Rental`, this should all happen in the `main.rb`.